### DISCUSSION TITLE
[Feature] Stable URLs for slideshows, photo displays, timeline

### DISCUSSION THREAD

**Discussion by jonhnet** [immich-app/immich#7017] [Feature Request]
### The feature

For kiosk-like applications, it's nice to be able to capture the state of the web view in the URL, so one can make it a bookmark that opens on start.

For example, if I could bookmark the URL of an open, displaying slideshow, then I can set that as the open-on-start bookmark on an Android dongle to turn a monitor into a digital picture frame.

Right now, when a slideshow is playing, the window URL is just the URL for the album. If I start a browser at that URL, it displays the album preview, which would require a user to click to launch the slideshow.

Likewise, if I search for an image and click it, the image fills the display (a static slideshow). I'd like to bookmark that view. Right now, the associated URL takes me back to the search results page.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
#8532

**Comment by jwlarocque** [comment]
This is a great update and appreciated, but from what I can tell doesn't actually address the feature request.  The URL now captures the image currently being displayed, but doesn't capture whether a slideshow is in progress, so it's not possible to for example use an open-on-start bookmark to create a digital picture frame.  Therefore I think the discussion could be reopened.
(Please correct me if I'm mistaken.)

**Reply by rbrtwillis** [reply]
I agree. I would like a way to set the URL directly to slide show mode for a given album. I use Fully Kiosk on an Android tablet. THanks.
